### PR TITLE
Migrate to consolidated secrets.conf file.

### PR DESCRIPTION
**Pull request by twaddington**
**What's this do?**
This updates the projects to use a consolidated `secrets.conf` file instead of individual config files in differing formats.

**Why are we doing this? (w/ JIRA link if applicable)**
Make configuring projects and managing secrets easier.

**How should this be tested? / Do these changes have associated tests?**
Ensure release builds compile correctly.

**Dependencies for merging? Releasing to production?**
Depends on https://github.com/NYPL-Simplified/Certificates/pull/16

**Has the application documentation been updated for these changes?**
Yes

**Did someone actually run this code to verify it works?**
Yes

### CODE REVIEW

**Comment by twaddington** [comment]
No rush on merging this one.

File: simplified-app-simplye/src/main/resources/META-INF/services/org.nypl.simplified.viewer.audiobook.AudioBookFeedbooksServiceType [hunk 1]
```diff
@@ -1 +0,0 @@
-org.nypl.simplified.simplye.FeedbooksConfigurationService
```
Comment (twaddington) [inline simplified-app-simplye/src/main/resources/META-INF/services/org.nypl.simplified.viewer.audiobook.AudioBookFeedbooksServiceType:1]: Looks like this service is unnecessary due to the `MainFeedbooksSecretService` in Core.

Unless I'm misunderstanding something?

Comment (io7m) [reply simplified-app-simplye/src/main/resources/META-INF/services/org.nypl.simplified.viewer.audiobook.AudioBookFeedbooksServiceType:1]: This looks like a vestigial organ. I think the service just got defined twice, but the app only looks at the first one it encounters anyway.

**Comment by io7m** [comment]
I'm fine with doing this.

**Pull request merged**